source/account.html.md

```diff
@@ -27,6 +27,7 @@ menu_items:
   - security
   - aviate-catalog-apis
   - aviate-health-apis
+  - aviate-metering-apis
```

Comment:
Can you remind me: Do we need to manage all these html files manually - or has this been generated somehow?

Reply:
I manually made changes to all the HTML files. I don't think there is a way to generate these files automatically.